src/hooks/shards/useShardsList.ts

```diff
@@ -40,18 +66,53 @@ const useShardsList = (catalogNames: string[]) => {
             return fetchShardList(name, session, reactorAuthorization);
         });
 
-        const shardResponses = await Promise.all(shardPromises);
+        let allCallsFailed = true;
+        const shardResponses = await Promise.allSettled(shardPromises);
+        shardResponses.forEach((shardResponse, index) => {
+            const catalogName = catalogsToFetch[index];
+
+            if (
+                isPromiseFulfilledResult(shardResponse) &&
+                shardResponse.value.shards.length > 0
+            ) {
+                allCallsFailed = false;
+                failureCountsRef.current[catalogName] &&= 0;
+                response.shards = response.shards.concat(
+                    shardResponse.value.shards
+                );
```

Comment:
If we fulfilled something we can mark that we got at least one success and then reset any counters if they exist